e2e/monorepo-support/test/monorepo_support/core_test.clj

```diff
@@ -22,7 +22,7 @@
 
     (doseq [filename all
             :let [file (File. filename)
-                  absolute (-> file .getAbsolutePath)]]
+                  absolute (-> file .getCanonicalPath)]]
```

Comment:
Settling in using `getCanonicalPath` only seems a good idea:

* one less thing to remember
* importantly, it resolves symlinks, resulting in cache keys that can hit more often

From memory, libs like tools.namespace favor .getCanonicalPath